### PR TITLE
Added Prizes details

### PR DESCRIPTION
**Pull request by Cassianky**
#104 
Added Prizes details
Change the length of other prizes in large screen

### CODE REVIEW

File: src/components/PrizesDetails/index.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            return (
+              <Grid key={index} item xs={12} sm={6}>
+                <ul>
+                  <li className={classes.listTitle}>
+                    <Typography className={classes.listTitle}>
+                      {p.prize}
+                    </Typography>
+                  </li>
+                  <div>
+                    <Typography variant="body1">{p.description}</Typography>
```
Comment (jerrenss) [inline src/components/PrizesDetails/index.jsx:56]: can remove variant prop since default is body1

**Review by jerrenss — COMMENTED** [review]
Thanks for the work! Looks good, will provide you the actual prize details to update soon!

**Comment by Cassianky** [comment]
ok thanks! I changed the minimum height to 100vh for the root as well.

**Comment by jerrenss** [comment]
Closing because there is no longer a need to display prize description. Thanks for the effort!

**Pull request closed without merging**